File: ThaiTuanERP2025.Client/src/app/pages/finance/models/ledger-account-type.dto.ts
export interface LedgerAccountTypeDto {
      id: string;
      code: string;
      name: string;
      ledgerAccountTypeKind: number;
      isActive: boolean;
      description?: string | null;
}

export interface LedgerAccountTypeRequest {
      code: string;
      name: string;
      ledgerAccountTypeKind: number;
      description?: string | null;
}

File: ThaiTuanERP2025.Server/ThaiTuanERP2025.Application/Finance/LedgerAccountTypes/Contracts/LedgerAccountTypeDto.cs
using ThaiTuanERP2025.Domain.Finance.Enums;

namespace ThaiTuanERP2025.Application.Finance.LedgerAccountTypes.Contracts
{
	public record LedgerAccountTypeDto
	{
		public Guid Id { get; init; }
		public string Code { get; init; } = string.Empty;
		public string Name { get; init; } = string.Empty;
		public LedgerAccountTypeKind LedgerAccountTypeKind { get; init; }
		public string? Description { get; init; } = string.Empty;
		public bool IsActive { get; init; }
	}
}
